test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1827,3 +1839,89 @@ UCS_TEST_SKIP_COND_P(test_ucp_address_v2, diff_seg_sizes,
 }
 
 UCP_INSTANTIATE_TEST_CASE(test_ucp_address_v2)
+
+class test_pci_bw : public test_ucp_wireup {
+public:
+    static void get_test_variants(std::vector<ucp_test_variant> &variants)
+    {
+        add_variant_with_value(variants, UCP_FEATURE_TAG, TEST_TAG, "tag");
+    }
+
+    std::string get_tcp_device(const std::string &dev_name)
```

Comment:
`auto`?

Reply:
replace std::string?

Reply:
If possible, why not?

Reply:
you mean for the function return type? seems it's not simple in c++ 11:
ucp/test_ucp_wireup.cc:1850:52: error: ‘get_tcp_device’ function uses ‘auto’ type specifier without trailing return type [-Werror]